doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -91,3 +91,151 @@ HAFS Workflow Schematic
 .. figure:: https://github.com/hafs-community/HAFS/wiki/docs_images/hafs_workflow_schematic.png
     :width: 75 %
     :alt: HAFS Workflow Schematic
+
+.. _OperationalUpdates:
+
+===============================
+HAFS v2.0.0 Operational Updates 
+===============================
+
+**Model:** Hurricane Analysis and Forecast System (HAFS)
+
+**Version:** hafs.v2.0.0
+
+**Released:** July 01, 2024, 1200UTC
+
+**Location on GitHub:** `HAFS GitHub Repository <https://github.com/hafs-community/HAFS/tree/production/hafs.v2>`
+
+**Clone the repository:**
+
+.. code-block:: bash
+
+   git clone -b production/hafs.v2 --recursive  https://github.com/hafs-community/HAFS.git ./hafs.v2.0.0
+
+**Purpose:**
+    Run HAFS.v2 HFSA configuration to provide hurricane track and intensity forecast guidance to 5.25 days over all global oceanic basins, including NATL, EPAC, CPAC, WPAC, NIO, and SH basins. Run HAFS.v2 HFSB configuration to provide hurricane track and intensity forecast guidance for 5.25 days over NHC/CPHC basins, including NATL, EPAC, CPAC basins.
+
+**Developed by:**
+    EMC and the UFS Hurricane Application Team. See currently active HAFS developers `here <https://docs.google.com/presentation/d/1xBNxvAG8-Kk3GS93PndaPVZp_L8U-KqGRGNhnviXZMg/edit?usp=sharing>`__
+    
+**Runs on:** WCOSS2
+
+Input/Output
+------------
+
+**Input:**
+    - Global Forecast System (GFS, including GFS/GDAS) NetCDF format analysis for atmospheric initial condition, and 3-hourly GRIB2 files (up to 129 h) for atmospheric lateral boundary conditions and atmospheric forcing for oceanic and wave model components.
+    - Global RTOFS for oceanic initial and boundary conditions for MOM6 and HYCOM.
+    - GFS/GDAS global wave analysis/forecast for HFSA wave component initial and lateral boundary conditions.
+    - OBSPROC observational data for HAFS data assimilation.
```

Comment:
Eventually, it would be good to link to an S3 bucket with all of this data, though that isn't possible now.